src/cmd5.cc

```diff
@@ -1879,7 +1879,8 @@ bool is_ok_spell(s32b spell_idx, s32b pval)
 	}
 	// Are we permitted to cast based on item pval? Only music
 	// spells have non-zero minimum PVAL.
-	if (pval < spell_type_minimum_pval(spell))
+	s32b min_pval = spell_type_minimum_pval(spell);
+	if (min_pval > 0 && pval < min_pval)
```

Comment:
This is where I'm a bit unusure... Should this be min_pval >= 0, or is > correct?

Reply:
Per the comment, only music spells have pvals != 0 and we only want to disable music spells so > is correct.